packages/app-store/getAppAssetFullPath.ts

```diff
@@ -0,0 +1,10 @@
+import type { App } from "@calcom/types/App";
+
+export function getAppAssetFullPath(assetPath: string, metadata: Pick<App, "dirName" | "isTemplate">) {
+  const appDirName = `${metadata.isTemplate ? "templates/" : ""}${metadata.dirName}`;
+  let assetFullPath = assetPath;
+  if (!assetPath.startsWith("/app-store/") && !/^https?/.test(assetPath)) {
```

Comment:
Now supports external resource URL as well which was the case with n8n DESCRIPTION.md file.